ui/pages/confirmations/components/confirm/footer/footer.test.tsx

```diff
@@ -202,7 +202,7 @@ describe('ConfirmFooter', () => {
     };
     it('renders the review alerts button when there are unconfirmed alerts', () => {
```

Comment:
isn't it the case to update the description to something like: `renders the confirm button when there is one unconfirmed alert`?